appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmRuntimeFileLogHandler.java

```diff
@@ -32,22 +32,20 @@
  * with the cloud logging agent.
  *
  */
-public class VmRuntimeFileLogHandler extends FileHandler {
-
+public class VmRuntimeFileLogHandler extends FileHandler implements SystemLogger {
```

Comment:
They could be, but I was leaving them in-place for backward compatibility reasons.